src/platform/graphics/render-pass.js

```diff
@@ -186,7 +186,7 @@ class RenderPass {
     }
 
     /**
-     * @param {import('../graphics/render-target.js').RenderTarget} renderTarget - The render
+     * @param {import('../graphics/render-target.js').RenderTarget|null} renderTarget - The render
```

Comment:
so yeah, here only the optional RenderTarget type should be provided as you suggested.

Reply:
Nice, changed this as discussed below here via `[renderTarget]` and `init(renderTarget = null)` in https://github.com/playcanvas/engine/pull/5704/commits/12272f20eb1954077bbbd9c5fe054fddfec9687c